onnx/defs/nn/defs.cc

```diff
@@ -1527,48 +1544,107 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Input(
             3,
             "mean",
-            "running (training) or estimated (testing) mean tensor of shape (C).",
+            "running (training) or fixed (testing) mean tensor of shape (C).",
```

Comment:
```suggestion
            "running (training) or estimated (testing) mean tensor of shape (C).",
```
I was wrong. The word `estimated` is more accurate. Sorry.